src/cmds.c

```diff
@@ -1946,17 +1941,24 @@ COMMAND(cmd_fetch)
     curl_easy_setopt(handle, CURLOPT_USERPWD, userpass);
     curl_easy_setopt(handle, CURLOPT_HTTPAUTH, CURLAUTH_ANY);
   }
+  
+  authorize = pe_regs_get(req->pe_regs, PE_REGS_Q, "authorization");
```

Comment:
```
&TEST #1=th %0; th [iter(listq(*),%i0: %q<%i0>,,%r)]

@http %!/test=[setq(authorization,[lit(bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwcmVmZXJyZWRfdXNlcm5hbWUiOiJyb290IiwiaXNzIjoiYXJhbmdvZGIiLCJpYXQiOjE2NjM2MzY1NDcsImV4cCI6MTY2MzY0MDE0N30.lU2tnbt1quVuKJLYocrcsASyQBB-ezch29uNn6pdbhY)])]http://localhost:8529/_api/collection
{"error":false,"code":200,"result":[{"id":"37","name":"_frontend","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_frontend"},{"id":"34","name":"_appbundles","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_appbundles"},{"id":"31","name":"_apps","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_apps"},{"id":"28","name":"_jobs","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_jobs"},{"id":"25","name":"_queues","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_queues"},{"id":"22","name":"_aqlfunctions","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_aqlfunctions"},{"id":"19","name":"_analyzers","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_analyzers"},{"id":"16","name":"_statisticsRaw","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_statisticsRaw"},{"id":"247","name":"_fishbowl","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_fishbowl"},{"id":"13","name":"_statistics15","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_statistics15"},{"id":"10","name":"_statistics","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_statistics"},{"id":"7","name":"_graphs","status":3,"type":2,"isSystem":true,"globallyUniqueId":"_graphs"}]}
AUTHORIZATION: bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwcmVmZXJyZWRfdXNlcm5hbWUiOiJyb290IiwiaXNzIjoiYXJhbmdvZGIiLCJpYXQiOjE2NjM2MzY1NDcsImV4cCI6MTY2MzY0MDE0N30.lU2tnbt1quVuKJLYocrcsASyQBB-ezch29uNn6pdbhY
CONTENT-TYPE: application/json; charset=utf-8
STATUS: 200
```

Reply:
Taking input on doing it this way, or allowing someone to set for instance...

```
setq(hdrs.authorization,value)
setq(hdrs.someother,value2)
```

Reply:
Changed from allowing an Authorization header to just allowing a body of headers. It contains a check to make sure you are not going to fail putting in headers in the right format, and strips ansi.